templates/vhost/_security.erb

```diff
@@ -5,6 +5,18 @@
 <% if @modsec_audit_log_destination -%>
   SecAuditLog "<%= @modsec_audit_log_destination %>"
 <% end -%>
+
+<% @modsec_before_crs.flatten.each do |rule| -%>
+<%= rule %>
+<% end -%>
+# ModSecurity Core Rules Set configuration
+<%- if scope.function_versioncmp([scope.lookupvar('::apache::apache_version'), '2.4']) >= 0 -%>
+IncludeOptional <%= @modsec_dir %>/*.conf
+IncludeOptional <%= @modsec_dir %>/activated_rules/*.conf
+<%- else -%>
+Include <%= @modsec_dir %>/*.conf
+Include <%= @modsec_dir %>/activated_rules/*.conf
```

Comment:
The 5 configuration line sshould be indented 2 spaces to match the rest of the config file.